users/models.py

```diff
@@ -7,23 +7,23 @@
 from django.contrib.auth.models import Permission
 
 
-
-
-
 @receiver(pre_save, sender=User)
 def set_group(sender, **kwargs):
     user = kwargs["instance"]
     if user.is_staff == 0:
         user.is_staff = 1
-        user.save()
     else:
         pass
 
+
 @receiver(post_save, sender=User)
 def set_permissions(sender, **kwargs):
     u, created = kwargs["instance"], kwargs["created"]
     if created:
-        permission = Permission.objects.get(codename='view_Case')
-        permission1 = Permission.objects.get(codename='view_letter')
+
+        permission = Permission.objects.get(content_type=1, codename='view_Case')
```

Comment:
Nie zakładałbym, że to będzie zawsze ```1```, bo przecież dodamy nową aplikacje w ```INSTALLED_APPS```, albo zmienimy tam kolejność i wartość będzie inna. Zobacz na https://github.com/django-guardian/django-guardian/blob/8382df9362dc80059864124e93d90418ae44637c/guardian/managers.py#L87-L88 i https://github.com/django-guardian/django-guardian/blob/4981fd22b5fb266ede53a40ef4f46c8671f2b8da/guardian/testapp/tests/test_direct_rel.py#L29-L30 jak pobrać dynamicznie taką wartość dla konkretnego ```ContentType``` (modelu). 

Poza tym dwa zapytania SQL ```SELECT``` do tak trywialnej rzeczy. Można to zrobić w ładny sposób w jednym zapytaniu, o ile - nie jestem pewien - czy nie można bezpośrednio wcielić do zapytania INSERT, ale aż tak nie musimy iść. Wystarczy jak zamiast dwa razy ```.get(...)``` wywołamy raz ```.all()```.